migrations/versions/7d344f2b41a9_add_uuid_to_user_table.py

```diff
@@ -0,0 +1,36 @@
+"""add uuid to user table
```

Comment:
You can delete this file now.

Reply:
We still need this in on the user table.